views/site/index.php

```diff
@@ -1 +1,52 @@
-Hello, world!
+<?php
+
+use Yiisoft\Yii\Bootstrap4\Carousel;
+
+echo Carousel::widget()
+    ->items([
+        [
+            'content' => '<div class="d-block w-100 bg-info" style="height: 200px"></div>',
+            'caption' => '<h5>First slide label</h5><p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>',
+            'captionOptions' => ['class' => ['d-none', 'd-md-block']],
+        ],
+        [
+            'content' => '<div class="d-block w-100 bg-secondary" style="height: 200px"></div>',
+            'caption' => '<h5>Second slide label</h5><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>',
+            'captionOptions' => ['class' => ['d-none', 'd-md-block']],
+        ],
+        [
+            'content' => '<div class="d-block w-100 bg-dark" style="height: 200px"></div>',
+            'caption' => '<h5>Third slide label</h5><p>Praesent commodo cursus magna, vel scelerisque nisl.</p>',
+            'captionOptions' => ['class' => ['d-none', 'd-md-block']],
+        ],
+    ]);
+?>
+
+
+
+<div class="card mt-3 col-md-6">
+    <div class="card-body">
+        <h2 class="card-title">Console</h2>
+        <?php $binPath = strtr('./vendor/bin/yii', '/', DIRECTORY_SEPARATOR); ?>
```

Comment:
There's no need to use directory separator here. It works well with forward slashes.

Reply:
No. There must be separators appropriate for the operating system.
In order for me to copy and run these commands on Windows, this panel should look like this:
![image](https://user-images.githubusercontent.com/4152481/74520592-f0797800-4f28-11ea-951e-a7a3b82e5b78.png)